src/main/java/seedu/address/logic/commands/DeleteCommand.java

```diff
@@ -48,7 +48,7 @@ public CommandResult execute(Model model) throws CommandException {
         model.deleteCandidate(candidateToDelete);
 
         return new CommandResult(String.format(MESSAGE_DELETE_CANDIDATE_SUCCESS, candidateToDelete), false,
-                false, true, -1, true, targetIndex.getZeroBased());
+                false, false, -1, true, targetIndex.getZeroBased());
```

Comment:
ello, just wanted to make a quick check as I'm a little unsure of what behaviour is impacted by the change of this from true to false?

Reply:
For this `boolean`, the argument is mainly used for the `FocusCommand`.  As for refreshing the `FocusCard`, it is further implemented in the `MainWindow` class and won't depend on this variable. 

I changed it to false to prevent any rouge `FocusCommand` from starting up (and will cause more exceptions)

Reply:
ah yep, that makes sense, got it, thank you!!